dali/pipeline/executor/queue_policy.h

```diff
@@ -178,24 +193,44 @@ struct UniformQueuePolicy {
   std::atomic<bool> ready_stop_ = {false};
 };
 
+struct SeparateQueuePolicy;
+
+struct ReleaseCommand {
```

Comment:
knitpick: wrap this in `namespace detail`

Reply:
Done